src/Bridge/Symfony/DependencyInjection/Compiler/MapperCompilerPass.php

```diff
@@ -37,8 +37,8 @@ public function process(ContainerBuilder $container): void
         $collector = DoctrineCollector::getInstance();
 
         foreach ($collector->getAssociations() as $class => $associations) {
-            foreach ($associations as $field => $options) {
-                $mapper->addMethodCall('addAssociation', [$class, $field, $options]);
+            foreach ($associations as $type => $options) {
+                $mapper->addMethodCall('addAssociation', [$class, $type, $options]);
```

Comment:
I would do this variable renaming in a separate PR

Reply:
Done